proto/Integration.test.ss

```diff
@@ -287,6 +287,16 @@ function eslint( test )
   // if( _.process.insideTestContainer() && process.platform !== 'linux' )
   // return test.true( true );
 
+  if( _.process.insideTestContainer() )
+  {
+    let majorVersion = process.env.npm_config_node_version.match( '/^\d+/' );
```

Comment:
must be better way

Reply:
Improved condition, used nodejs variables, but any context variable such as ${{ matrix }} is available only in workflow file.